packages/salesforcedx-vscode-core/src/util/configUtil.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ConfigAggregator, ConfigFile, ConfigValue } from '@salesforce/core';
+import * as path from 'path';
+import { isNullOrUndefined, isUndefined } from 'util';
+import { getRootWorkspacePath } from './index';
+
+export enum ConfigSource {
+  Local,
+  Global,
+  None
+}
+
+// This class should be reworked or removed once the ConfigAggregator correctly checks
```

Comment:
Maybe we can include this info in the chatter for the bug instead?

Reply:
I want to leave the comments in here. The reason being is that we don't control that class and we don't actually know when it'll be fixed. Further, it states intent. I'd had to ask 2 different people (both you and @lcampos) why we weren't using the ConfigAggregator to begin with because there were no comments in the code indicating why processing had to create the local config and then use the ConfigAggregator for the global.